Apps/Laboratory/src/appsettings.hgpoc.json

```diff
@@ -10,6 +10,15 @@
     "Laboratory": {
         "EndPoint": "https://phsahealthgatewayapi-dev.azurewebsites.net/api/v1/LabOrders"
     },
+    "PatientService": {
```

Comment:
Nice of you to maintain this file!

Reply:
if its there we should maintain it, otherwise lets delete it